go-runtime/scaffolding/{{ .Name | camel | lower }}/ftl.toml

```diff
@@ -1,3 +1,4 @@
 module = "{{ .Name }}"
 language = "go"
-deploy = ["build/main", "build/schema.pb"]
+deploy-dir = "build"
+deploy = ["main", "schema.pb"]
```

Comment:
@alecthomas not sure if we want to move `schema.pb` out of the `deploy` section and into `schema = "schema.pb"` like the kotlin example.